src/integration/charging-station-vendor/sap/SAPChargingStationVendorIntegration.ts

```diff
@@ -0,0 +1,8 @@
+import ChargingStation from '../../../types/ChargingStation';
+import ChargingStationVendorIntegration from '../ChargingStationVendorIntegration';
+
+export default class SAPChargingStationVendorIntegration extends ChargingStationVendorIntegration {
```

Comment:
Why not Shelly instead of SAP.
SAP does not make chargers.

Reply:
Shelly does not make the charger. It just provides a socket with a relay. The firmware is developed by us and the wallbox is built here.
The vendor name is SAP, this is the reason why I used SAPChargingStationVendorIntegration for the class. 

I can rename it if you prefer.

It would be great to have a default vendor integration which follows the standard to avoid empty integration like this one.

Reply:
Ok I understand so I would prefer that you use SAP Labs France instead ;-)
Thanks.

Reply:
Done